src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/BotSegments/Models/BotSequence.cs

```diff
@@ -40,7 +40,7 @@ public class BotSequence
          */
         public string resourcePath;
 
-        public static BotSequence ActiveBotSequence = null;
+        public static volatile BotSequence ActiveBotSequence = null;
```

Comment:
This doesn't strictly need to be volatile for this PR's fixes, but I went ahead and changed it since this field is managed in other code that may run on a web request callback.